### ISSUE TITLE
Should `normalise` switch to the uncorrected standard deviation?

### ISSUE THREAD

**Issue by jekbradbury** [FluxML/Flux.jl#529]
The current implementation of `Flux.normalise` (used in `LayerNorm`) calls `Statistics.std`, which defaults to using corrected variance (dividing the sum of squared residuals by N-1 rather than N).  However, all external layer norm implementations I'm familiar with (the built-in PyTorch implementation and two popular TensorFlow implementations) use the uncorrected variance, even if their "standard deviation" functions use the corrected variance. And while Flux doesn't use normalise for batchnorm, it does rely on libraries (like cuDNN) that compute batchnorm with the uncorrected variance.

Should we switch outright, or add a flag?

**Comment by MikeInnes** [comment]
I'm fine with switching outright.
